src/base.lib.php

```diff
@@ -580,6 +580,77 @@ public static function supportReverseProxy()
         return SucuriScanOption::isEnabled(':revproxy');
     }
 
+    /**
+     * Checks whether an IPv4 address is in a CIDR range.
+     *
+     * @param string $ip Valid IPv4 address.
+     * @param string $range Valid CIDR range.
+     * @return boolean True if the IP is in the CIDR range.
+     */
+    public static function is_ipv4_addr_in_range($ip = '', $range = '')
```

Comment:
Not a big deal, but most of the plugins functions are using a camel case style for naming. Same for "inet_to_bytes" and "is_ipv6_addr_in_range" functions.

Reply:
Since you don't expect to receive empty $ip and $range, you should remove the default empty values.